build-constraints.yaml

```diff
@@ -6445,343 +7525,760 @@ packages:
+        - servant-yaml < 0 # tried servant-yaml-0.1.0.1, but its *library* does not support: base-4.16.0.0
+        - servant-yaml < 0 # tried servant-yaml-0.1.0.1, but its *library* does not support: bytestring-0.11.1.0
         - servant-yaml < 0 # tried servant-yaml-0.1.0.1, but its *library* does not support: servant-0.18.3
         - serverless-haskell < 0 # tried serverless-haskell-0.12.6, but its *library* requires the disabled package: amazonka-core
+        - serversession < 0 # tried serversession-1.0.2, but its *library* requires the disabled package: persistent-test
         - serversession-backend-persistent < 0 # tried serversession-backend-persistent-1.0.5, but its *library* does not support: base64-bytestring-1.2.1.0
         - serversession-backend-redis < 0 # tried serversession-backend-redis-1.0.4, but its *library* does not support: hedis-0.15.0
+        - serversession-frontend-wai < 0 # tried serversession-frontend-wai-1.0, but its *library* requires the disabled package: serversession
+        - servius < 0 # tried servius-1.2.3.0, but its *executable* requires the disabled package: shakespeare
+        - ses-html < 0 # tried ses-html-0.4.0.0, but its *library* requires the disabled package: cryptohash
+        - ses-html < 0 # tried ses-html-0.4.0.0, but its *library* requires the disabled package: http-streams
+        - sessiontypes-distributed < 0 # tried sessiontypes-distributed-0.1.1, but its *library* does not support: bytestring-0.11.1.0
         - sessiontypes-distributed < 0 # tried sessiontypes-distributed-0.1.1, but its *library* does not support: exceptions-0.10.4
         - sessiontypes-distributed < 0 # tried sessiontypes-distributed-0.1.1, but its *library* requires the disabled package: sessiontypes
-        - sexpr-parser < 0 # tried sexpr-parser-0.2.0.0, but its *library* does not support: base-4.15.0.0
+        - sexp-grammar < 0 # tried sexp-grammar-2.3.3.1, but its *library* requires the disabled package: invertible-grammar
+        - sexpr-parser < 0 # tried sexpr-parser-0.2.0.0, but its *executable* does not support: bytestring-0.11.1.0
+        - sexpr-parser < 0 # tried sexpr-parser-0.2.0.0, but its *library* does not support: base-4.16.0.0
         - sexpr-parser < 0 # tried sexpr-parser-0.2.0.0, but its *library* does not support: megaparsec-9.2.0
+        - shake-language-c < 0 # tried shake-language-c-0.12.0, but its *library* requires the disabled package: fclabels
+        - shake-plus < 0 # tried shake-plus-0.3.4.0, but its *library* requires the disabled package: path
         - shake-plus-extended < 0 # tried shake-plus-extended-0.4.1.0, but its *library* requires the disabled package: ixset-typed
+        - shake-plus-extended < 0 # tried shake-plus-extended-0.4.1.0, but its *library* requires the disabled package: path
+        - shakespeare < 0 # tried shakespeare-2.0.25, but its *library* does not support: aeson-2.0.2.0
```

Comment:
This was already fixed, just waiting for a release: https://github.com/yesodweb/shakespeare/pull/260